mlflow/tracking/client.py

```diff
@@ -1349,15 +2019,130 @@ def delete_model_version(self, name, version):
 
         :param name: Name of the containing registered model.
         :param version: Version number of the model version.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow.sklearn
+            from mlflow.tracking import MlflowClient
+            from sklearn.ensemble import RandomForestRegressor
+
+            def print_models_info(mv):
+                for m in mv:
+                    print("name: {}".format(m.name))
+                    print("latest version: {}".format(m.version))
+                    print("run_id: {}".format(m.run_id))
+                    print("current_stage: {}".format(m.current_stage))
+
+            mlflow.set_tracking_uri("sqlite:///mlruns.db")
+
+            # Create two runs and log MLflow entities
+            with mlflow.start_run() as run1:
+                params = {"n_estimators": 3, "random_state": 42}
+                rfr = RandomForestRegressor(**params)
+                mlflow.log_params(params)
```

Comment:
```suggestion
                rfr = RandomForestRegressor(**params).fit([[0, 1]], [2])
                mlflow.log_params(params)
```

Can we call `fit` using toy data to prevent users from wondering "why fit isn't called?" + make the example more realistic?

Reply:
In this example, it doesn't matter whether the model is fitted or not though.

Reply:
We use this `rfr = RandomForestRegressor(**params)` model in many API calls here. Arguably, if we call `rfr`.fit()` here, we might as well make it ubiquitous elsewhere, for the same argument can be used there too, especially with "create_model_* etc. WDYT?

Reply:
Let's just leave `rfr` unfitted in this PR.

Reply:
And update them all later in a separate PR.

Reply:
I like the idea of having the `fit` call in other places too. Let's go with your suggestion of filling another PR where we retrofit this `fit` call.